kernel/README.md

```diff
@@ -1,13 +1,13 @@
 # Build Kata Containers Kernel
 
 This document explains the steps to build a compatible kernel with Kata
-Containers. To do this use build-kernel.sh, this script automates the
```

Comment:
maybe /to build a compatible kernel with Kata/to build a kernel recommended for use with Kata Containers/?